TESTS/mbed_hal/common_tickers/main.cpp

```diff
@@ -563,13 +579,16 @@ utest::v1::status_t lp_ticker_teardown(const Case *const source, const size_t pa
     ticker_resume(get_lp_ticker_data());
     osKernelResume(0);
 
+    /* Redeem from missing go-off ticker interrupt (same as us ticker above) */
+    lp_ticker_fire_interrupt();
```

Comment:
This call to us_ticker_fire_interrupt should no longer be needed with the fix #7524. The call to `ticker_resume` earlier in this function should restart the common ticker layer.

Reply:
@c1728p9 Fixed as above.